taipy/gui/utils/_evaluator.py

```diff
@@ -89,6 +89,19 @@ def _is_expression(self, expr: str) -> bool:
     def _fetch_expression_list(self, expr: str) -> t.List:
         return [v[0] for v in _Evaluator.__EXPR_RE.findall(expr)]
 
+    def _get_variable_dependencies(self, var_name: str) -> t.Set[str]:
+        var_set: t.Set[str] = set()
+        for expr in self.__var_to_expr_list.get(var_name, []):
+            var_map = self.__expr_to_var_map.get(expr, {})
+            if len(var_map) == 1 and var_name in var_map:
```

Comment:
Please discuss that with me...

Reply:
len(var_map) > 1 means this is an expression which deal with our variable and another one